core/comment.ts

```diff
@@ -207,9 +204,7 @@ export class Comment extends Icon {
    * Resize the text area accordingly.
    */
   private onBubbleResize_() {
-    if (!this.isVisible()) {
-      return;
-    }
+    if (!this.bubble_) return;
```

Comment:
Why switch from `isVisible()` to `bubble_` for the check?

Reply:
So typescript can know that `this.bubble_` is defined for the `this.bubble_.getBubbleSize()` call